code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -195,7 +195,7 @@
 
 /obj/item/reagent_containers/food/snacks/pizzaslice/arnold
 	name = "\improper Arnold pizza slice"
-	desc = "I come over, maybe I give you a pizza, maybe I break off your arm."
+	desc = "I come over, maybe I give you a pizza, maybe I break off your arm. <span class='warning'>This will probably be bad for you...</span>"
```

Comment:
```suggestion
	desc = "I come over, maybe I give you a pizza, maybe I break off your arm. <span class='warning'>Eating this will not give you a pizza...</span>"
```